docs/pages/admin-guide.mdx

```diff
@@ -827,18 +765,18 @@ stored in `data_dir` under `log` directory, usually `/var/lib/teleport/log` .
 Each day is represented as a file:
 
 ```bash
-$ ls -l /var/lib/teleport/log/
-total 104
--rw-r----- 1 root root  31638 Jan 22 20:00 2017-01-23.00:00:00.log
--rw-r----- 1 root root  91256 Jan 31 21:00 2017-02-01.00:00:00.log
--rw-r----- 1 root root  15815 Feb 32 22:54 2017-02-03.00:00:00.log
+ls -l /var/lib/teleport/log/
+# total 104
```

Comment:
I've uncommented all output examples and added a space, a comment stating `# Output` to separate the output from the command. Along with a note `{/* Convert to new UI component https://github.com/gravitational/next/issues/275 */}` so we can GREP and find all these.